components/drivers/usb/usbdevice/class/audio_mic.c

```diff
@@ -160,7 +161,11 @@ static struct uac_ac_descriptor ac_desc =
         USB_CLASS_AUDIO,
         USB_SUBCLASS_AUDIOCONTROL,
         0x00,
-        0x00,
+#ifdef RT_USB_DEVICE_COMPOSITE
+		MIC_INTF_STR_INDEX,
```

Comment:
Please replace all tabs with spaces

Reply:
okay, done. subitted another PR " [component][driver][usb] Make compositive usb device string more clear" for this review